CONTRIBUTING.adoc

```diff
@@ -11,7 +12,31 @@ submit your pull request.
 If you have any questions about a possible submission, feel free to open
 an issue too.
 
-== Contributing to the terraform-oci-oke repository
+== Coding conventions
+
+Be sure to check {uri-coding-conventions}[the coding conventions] of this project before starting to develop.
+
+== Documentation you need to update
+
+A code contribution is as valuable as the related documentation that comes with it.
+
+Please update the documentation for the code you propose. The maintainers will love you, and your contribution will probably be accepted faster than if there is no or missing documentation.
+
+This is the actual "touch points" you need to pay attention:
+
+- Update CHANGELOG under the 'unreleased' section. Maintainer will reaffect your entry to another section in the release process
```

Comment:
Link to CHANGELOG file added, as well as links to other files mentioned in the documentation.

As there is a mix of *.md and *.adoc files for the documentation, I would suggest to not mention extension for common documentation files (CHANGELOG, CONTRIBUTING, CONTRIBUTORS): it will require less effort if we decide later to change documentation format (probably a new issue to create: single documentation format).